table/tables/cache_test.go

```diff
@@ -15,11 +15,16 @@
 package tables_test
 
 import (
+	"context"
 	"testing"
 	"time"
 
+	"github.com/pingcap/tidb/parser/model"
+	table2 "github.com/pingcap/tidb/table"
```

Comment:
Why rename it, is there a conflict name `table`?

Reply:
sorry. my mistake